src/main/java/careconnect/model/person/Person.java

```diff
@@ -76,8 +75,8 @@ public Set<Tag> getTags() {
      * Returns an immutable log list, which throws {@code UnsupportedOperationException}
      * if modification is attempted.
      */
-    public List<Log> getLogs() {
-        return Collections.unmodifiableList(this.logs);
+    public ArrayList<Log> getLogs() {
+        return new ArrayList<>(Collections.unmodifiableList(this.logs));
```

Comment:
should update the docs to indicate, returns a true copy of the logs. Modifying the true copy won't affect the original logs